src/cycax/main.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
```

Comment:
This looks like an example to me. Is that correct? If it is the case, thanks examples are great. Please put the example outside of the Library code.  We dont want to ship examples inside the library.
I think src/examples or example would be a good place.
The example does not have to be called main, I want to call the main scripts main.py in the Tsolo CAD repo.